packages/common/src/discussions/_internal/DiscussionBusinessRules.ts

```diff
@@ -42,7 +42,7 @@ export const DiscussionPermissionPolicies: IPermissionPolicy[] = [
     permission: "hub:discussion:view",
     dependencies: ["hub:discussion"],
     authenticated: false,
-    licenses: ["hub-premium"],
```

Comment:
Though doesn't seem to impact any functionality today, expanding to include `hub-basic` as those users are technically permitted to view discussion boards.